app/models/caf.rb

```diff
@@ -33,6 +35,8 @@ class Caf < ApplicationRecord
 
   scope :actives, -> { where(actived: true) }
```

Comment:
@radinreth pleases check whether CAF has been deleted but seems status still remain "active" -> so active will remain include soft-deleted CAF, please confirm

Reply:
Once CAF is soft-deleted, the status is meaningless, 
but do you want to update the status to inactive after soft delete right?

Reply:
@radinreth please check in the scope :actives, -> { where(actived: true) } line above, for the user perspective, when CAFs is being soft-deleted -> Caf.actives should not be included those are soft-deleted, does it make sense?

Reply:
right bong, but `act_as_paranoid` does this for us, we don't have to care about it

```sql
Caf.actives.to_sql
SELECT \"cafs\".* FROM \"cafs\" WHERE \"cafs\".\"deleted_at\" IS NULL AND \"cafs\".\"actived\" = TRUE
```